errors/error_test.go

```diff
@@ -65,24 +80,40 @@ func TestSkipWorks(t *testing.T) {
 	a()
 }
 
+func checkFramesMatch(expected StackFrame, actual StackFrame) bool {
```

Comment:
This one might be deceptive. Printing each time there's a mismatch is verbose, since there are differing system frames, etc, and those are fine. That's why the `assertStacksMatch()` function (the only caller) handles the assertion and printing the whole stack at once.